how-to/virtualisation/qemu.md

```diff
@@ -35,6 +60,147 @@ qemu-system-x86_64 options image[s]
 
 So take a look at the [QEMU manpage](http://manpages.ubuntu.com/manpages/bionic/man1/qemu-system.1.html), [`qemu-img`](http://manpages.ubuntu.com/manpages/bionic/man1/qemu-img.1.html) and the [QEMU documentation](https://www.qemu.org/documentation/) and see which options best suit your needs.
 
-## Next steps
+While a standard QEMU configuration works for most use cases, some scenarios demand high-vCPU VMs. In the next section, we’ll cover how to create QEMU virtual machines with up to 1024 vCPUs.
+
```

Comment:
It might be a good idea to keep the anchors from the original pages, above the original titles. In many cases this should ensure that any links to those original anchors don't get broken (any that are will be picked up by the redirects).